app/controllers/merchant/dashboard_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class Merchant::DashboardController < Merchant::BaseController
-  def index; end
+  def index
```

Comment:
Happy to change this before merging to a #show action instead of an #index action.

Reply:
Show seems more appropriate.

Reply:
This is for user story 30 correct? It says that the dashboard should show a list of all pending orders. A list of orders would indicate an index.

Reply:
This is user story 29

Reply:
Interesting. User story 29 and user story 30 both route to '/merchant'. #29 says the page needs to display the merchant information, and #30 says it should list all pending orders. It appears that the same page will need to render both sets of information. Being such, I think you can argue calling it either show or index. Whatever the group wants to do.

Reply:
And I see I used the number indicators incorrectly. Story 29 is #14 and story 30 is #13.